### PR TITLE
Add previous_meetings.html page

### PR DESCRIPTION
**Pull request by jnewbery**
Also adds links to 'home' and 'previous_meetings' in the header.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -3,8 +3,9 @@
     <a href="{{ site.github.url }}/" class="logo">{{ site.data.settings.title }}</a>
   </div>
   <nav>
+    |
     {% for item in site.data.settings.menu %}
-      <a href="{{ site.github.url }}/{{ item.url }}">{{ item.name }}</a>
+      <a href="{{ site.github.url }}/{{ item.url }}">{{ item.name }}</a> |
```
Comment (jonatack) [inline _includes/header.html]: nit: no space needed after | at the end of this line, or alternatively can place `|` by itself on the following line.

File: index.html [hunk 2]
```diff
@@ -35,7 +35,7 @@ <h2 class="Home-posts-title">Upcoming meetings</h2>
     <p>We're always looking for interesting PRs to discuss in review club and for volunteer hosts to lead the discussion. To suggest a PR or to offer to host a meeting, please leave a comment on <a href="https://github.com/bitcoin-core-review-club/bitcoin-core-review-club.github.io/issues/14">this github issue</a>.
 
     <h2 class="Home-posts-title">Previous meetings</h2>
-    {% for post in site.posts %}
+    {% for post in site.posts limit:5 %}
```
Comment (jonatack) [inline index.html]: nit: space between `limit:` and `5`.

File: _includes/header.html (hunk 1 above)
Comment (jnewbery) [reply _includes/header.html]: fixed

File: index.html (hunk 2 above)
Comment (jnewbery) [reply index.html]: fixed

**Comment by jnewbery** [comment]
Thanks for the review @jonatack . I've applied your two suggested changes.

**Pull request merged**